index.rst

```diff
@@ -136,10 +136,10 @@ by the appropriate change control body:
 Typically, adjustments to Level 1, 2 and 3 milestones are made every 6 or 12 months. Changes to Level 4
 milestones can occur more frequently.
 
-Short-term Planning
+Short-Term Planning
```

Comment:
If it's part of a title or a proper name (as opposed to just the start of a sentence), and it's not a "small" word like "and", a word after a hyphen should be capitalized.